### PR TITLE
SCRUM-4305 set obsolete / internal variables explicitly when being up…

### PR DESCRIPTION
**Pull request by cmpich**
…dated. (#1629)

(cherry picked from commit bc8af6e7164f7ec3870915b904ea46e9aa1c2d2c)

### CODE REVIEW

File: src/main/java/org/alliancegenome/curation_api/services/helpers/UniqueIdentifierHelper.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+import org.alliancegenome.curation_api.model.entities.Annotation;
+import org.alliancegenome.curation_api.model.entities.base.AuditedObject;
+import org.alliancegenome.curation_api.model.entities.base.SubmittedObject;
+import org.alliancegenome.curation_api.model.ingest.dto.AnnotationDTO;
+import org.alliancegenome.curation_api.model.ingest.dto.DiseaseAnnotationDTO;
+import org.alliancegenome.curation_api.model.ingest.dto.base.SubmittedObjectDTO;
+import org.apache.commons.lang3.StringUtils;
+
+public class UniqueIdentifierHelper {
```
Comment (devopsagr) [inline src/main/java/org/alliancegenome/curation_api/services/helpers/UniqueIdentifierHelper.java:11]: 🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.design.HideUtilityClassConstructorCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Utility classes should not have a public or default constructor.

File: src/main/java/org/alliancegenome/curation_api/services/helpers/UniqueIdentifierHelper.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+	private UniqueIdentifierHelper(){};
```
Comment (devopsagr) [inline src/main/java/org/alliancegenome/curation_api/services/helpers/UniqueIdentifierHelper.java:13]: 🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'{' is not followed by whitespace.

Comment (devopsagr) [inline src/main/java/org/alliancegenome/curation_api/services/helpers/UniqueIdentifierHelper.java:13]: 🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'{' is not preceded with whitespace.

Comment (devopsagr) [inline src/main/java/org/alliancegenome/curation_api/services/helpers/UniqueIdentifierHelper.java:13]: 🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'}' is not preceded with whitespace.

File: src/main/java/org/alliancegenome/curation_api/services/helpers/UniqueIdentifierHelper.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+import org.alliancegenome.curation_api.model.entities.base.AuditedObject;
+import org.alliancegenome.curation_api.model.entities.base.SubmittedObject;
+import org.alliancegenome.curation_api.model.ingest.dto.AnnotationDTO;
+import org.alliancegenome.curation_api.model.ingest.dto.DiseaseAnnotationDTO;
+import org.alliancegenome.curation_api.model.ingest.dto.base.SubmittedObjectDTO;
+import org.apache.commons.lang3.StringUtils;
+
+public class UniqueIdentifierHelper {
+
+	private UniqueIdentifierHelper(){
```
Comment (devopsagr) [inline src/main/java/org/alliancegenome/curation_api/services/helpers/UniqueIdentifierHelper.java:13]: 🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'{' is not preceded with whitespace.

**Review by oblodgett — APPROVED** [review]

**Pull request merged**